preprocessing/berlin/compile-berlin-geojson.js

```diff
@@ -106,6 +109,20 @@ function prepareFeatureProperties(feature) {
 	return outputProperties;
 }
 
+/*
+ * Returns a location text composed from given input parameters.
+ */
+function getLocationText(streetNameAndNumber, zipCode) {
+	var location = "";
+	if (streetNameAndNumber !== undefined && streetNameAndNumber !== "") {
+		location = streetNameAndNumber;
+	}
+	if (zipCode !== undefined && zipCode !== "") {
+		location += ", " + zipCode;
+	}
+	return location;
+}
```

Comment:
If `streetNameAndNumber` is undefined but `zipCode` is defined the function retunes something like: ", 90766"

```suggestion
function getLocationText(streetNameAndNumber, zipCode) {
  return [streetNameAndNumber, zipCode].filter(t => t && t.length).join(', ');
}
```

Reply:
Thanks. I will change this.

Reply:
@mojoaxel Do you like to prepare a pull request for your suggested change? If so, please re-run the preprocessor with the Berlin data to see if something changes.

Reply:
@mojoaxel Ping.

Reply:
@mojoaxel Feel free to change this. I won't have time the next weeks myself.

Reply:
:heavy_check_mark: Address here: https://github.com/wo-ist-markt/wo-ist-markt.github.io/pull/335.